src/ui/checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,10 @@
+import * as Styled from './Checkbox.styles';
+import { CheckboxProps } from './Checkbox.types';
+
+export const Checkbox = (props: React.ComponentPropsWithoutRef<'input'> & CheckboxProps) => (
```

Comment:
Dobrze by było przerzucić `React.ComponentPropsWithoutRef<'input'>` do `CheckboxProps`:
```
export type CheckboxProps = React.ComponentPropsWithoutRef<'input'> & {
...
```
To spowoduje, że te propsy rzeczy związane z typami będą w pliku z typami